src/chart/options/channel.cpp

```diff
@@ -89,10 +89,10 @@ bool Channel::isSeriesUsed(const Data::SeriesIndex &index) const
 	    || (dimensionIds.includes(index));
 }
 
-int Channel::findPos(const Data::SeriesIndex &index) const
+std::optional<int> Channel::findPos(const Data::SeriesIndex &index) const
```

Comment:
findPos in Channel and Channels are not used, can be removed.

Reply:
ok